### ISSUE TITLE
Add Hyperlink to Event Detail URLs

### ISSUE THREAD

**Issue by c3bryant** [UCSD/campus-mobile#428]
This task is to add a hyperlink to any http/s URLs found in the event description field on the event detail view.

![events-hyperlink.png](https://images.zenhubusercontent.com/5a7c89504b5806bc2bc62c34/ca46209c-bb12-446d-bab9-dc744a5f2a7e)

**Comment by vishant101** [comment]
![image.png](https://images.zenhubusercontent.com/5c533c9683ccb62a1a08b582/35a7f772-1a73-4580-b812-00b56a212607)

**Comment by c3bryant** [comment]
Android and iOS failed with one issue:

1) The clickable link area appears to be above the link, making it difficult to click on. The area outlined in red is where the actual clickable area seems to be:

![Screen Shot 2019-02-13 at 11.34.04 AM.png](https://images.zenhubusercontent.com/5a7c89504b5806bc2bc62c34/700155af-8435-4177-992f-44156047cd51)

As mentioned in issue #406, hyperlinks on the Notifications tab for Android have passed testing whereas here they have not.

**Comment by vishant101** [comment]
@c3bryant I've identified the cause of this issue and have implemented a fix.
 
Essentially the  `<Hyperlink>`  tag encapsulates the `<Text>` tag. What actually happens is the hyperlink view is drawn first, including the clickable area for the hyperlink. Then the text is drawn. Now if the text is styled with formatting that includes padding it means that the text being drawn no longer lines up with the hyperlink that has already been drawn. 

The solution I've implemented removes the padding from the `<Text>` tag and moves it to a `<View>` tag  that encapsulates the `<Hyperlink>`. This preserves the styling of the description and fixes the bug.
